migrations/20200526095359-create-program-for-observations.js

```diff
@@ -1,98 +1,203 @@
+const { ObjectID } = require("mongodb");
+
 module.exports = {
   async up(db) {
     
     global.migrationMsg = "Programs added for observations and observation submissions";
     
+    // Dependencies
+    const csv = require("csvtojson");
+    const fs = require("fs");
+    let path = "/observationSolutionToProgramMapping.csv";
```

Comment:
@aman-tunerlabs - This should be in project root not in system or machine root path